skrf/tests/test_network.py

```diff
@@ -597,6 +598,52 @@ def test_noise(self):
 
         return
 
+    def test_noise_deembed(self):
+        
+        if False :
+            ntwk4_n = rf.Network('ntwk4_n.s2p')['75-75.05GHz']
+            ntwk4 = rf.Network('ntwk4.s2p')['75-75.05GHz']
+            thru = rf.Network('thru.s2p')['75-75.05GHz']
+
+        ntwk4_n = rf.Network(os.path.join(self.test_dir,'ntwk4_n.s2p'))['75-75.05GHz']
+        ntwk4 = rf.Network(os.path.join(self.test_dir,'ntwk4.s2p'))['75-75.05GHz']
+        thru = rf.Network(os.path.join(self.test_dir,'thru.s2p'))['75-75.05GHz']
+        
+        ntwk4_thru = ntwk4 ** thru
+        retrieve_thru =  ntwk4.inv ** ntwk4_thru
+
+
+        self.assertEqual(retrieve_thru, thru)
+        self.assertTrue(ntwk4_thru.noisy)
+        self.assertTrue(retrieve_thru.noisy)
+        self.assertTrue(abs(thru.nfmin[0] - retrieve_thru.nfmin[0]) < 1.e-6, 'nf not retrieved by noise deembed')
+        self.assertTrue(abs(thru.rn[0]    - retrieve_thru.rn[0])    < 1.e-6, 'rn not retrieved by noise deembed')
+        self.assertTrue(abs(thru.z_opt[0] - retrieve_thru.z_opt[0]) < 1.e-6, 'noise figure does not match original spec')
+
+        ntwk4_n_thru = ntwk4_n ** thru
```

Comment:
When I de-embed on the load side using the following code to replace these lines, the tests below fail. Is right-side 'inverse connect' not supported?
```
ntwk4_n_thru = thru ** ntwk4_n
retrieve_n_thru =  ntwk4_n_thru ** ntwk4_n.inv
```